### PR TITLE
added exception handling for auto class

### PR DESCRIPTION
**Pull request by Scott771**

### CODE REVIEW

File: tests/auto_class/test_auto_classifier.py [hunk 1]
```diff
@@ -30,11 +30,26 @@ def test_create_classifier(indico_client, testdir_file_path):
...
+        assert df.shape[1] == 3
+        assert "a" in df["text"].tolist()
+
+
 def test_create_classifier_too_few_classes(indico_client, testdir_file_path):
     aclass = AutoClassifier(
-        indico_client,
-        os.path.join(testdir_file_path, "data/auto_class/class_a")
-        )
+        indico_client, os.path.join(testdir_file_path, "data/auto_class/class_a")
```
Comment (chris-wells-13) [inline tests/auto_class/test_auto_classifier.py:51]: does this need to be black-ed?

File: indico_toolkit/auto_class/classifier.py [hunk 2]
```diff
@@ -46,7 +51,13 @@ def create_classifier(self, verbose: bool = True, batch_size: int = 5):
         for i, fpaths in enumerate(self._fp.batch_files(batch_size=batch_size)):
             if verbose:
                 print(f"Starting batch {i + 1} of {len(self.file_paths) // batch_size}")
-            self.file_texts.extend(self.run_ocr(fpaths, text_setting="full_text"))
+            try:
+                self.file_texts.extend(self.run_ocr(fpaths, text_setting="full_text"))
+            except Exception as e:
+                if not catch_exceptions:
+                    raise e
+                self.file_texts.extend([None for i in range(batch_size)])
```
Comment (chris-wells-13) [inline indico_toolkit/auto_class/classifier.py:59]: this hurts a little.  we lose the whole batch if one file errors out, right?

**Review by chris-wells-13 — APPROVED** [review]
is there anything we can do to avoid losing an entire batch if we have just one bad file in it?

File: tests/auto_class/test_auto_classifier.py (hunk 1 above)
Comment (Scott771) [reply tests/auto_class/test_auto_classifier.py:51]: it got blacked (that's where the change on this came from)

**Pull request merged**